packages/core/frontend/sirius-components-core/src/workbench/Workbench.types.ts

```diff
@@ -37,12 +37,17 @@ export interface WorkbenchViewContribution {
   side: WorkbenchViewSide;
   title: string;
   icon: React.ReactElement;
-  component: (props: WorkbenchViewComponentProps) => JSX.Element | null;
+  component: ForwardRefExoticComponent<
+    PropsWithoutRef<WorkbenchViewComponentProps> & RefAttributes<WorkbenchViewHandle>
+  >;
+  ref: MutableRefObject<WorkbenchViewHandle | null> | undefined;
+  createRef: () => MutableRefObject<WorkbenchViewHandle | null>;
```

Comment:
I haven't reviewed everything but I feel that there is an easier way to type this, it's a bit odd to have something this complex. I'll look into it